src/components/SliderBox/mocks/slidesData.ts

```diff
@@ -0,0 +1,22 @@
+interface SlidesType {
+  name: string;
+  date: string;
+}
+
+export const slides: Array<SlidesType> = [
+  {
+    name: 'Upcoming',
+    date: '2022-04-13T19:28:34Z',
+  },
+  {
+    name: 'After Upcoming',
+    date: '2022-05-05T19:28:34Z',
+  },
+  {
+    name: 'Previous',
+    date: '2022-04-06T19:28:34Z',
+  },
+];
+
+export const imageSliderUrl =
+  'https://upload.wikimedia.org/wikipedia/commons/d/d1/Falcon_Heavy_Demo_Mission_%2839337245145%29.jpg';
```

Comment:
why do not you use this url in slides const? I guess launches should have different inages

Reply:
fixed